koji-scm-lib/src/main/java/org/fakekoji/xmlrpc/server/xmlrpcresponse/PackageId.java

```diff
@@ -0,0 +1,24 @@
+package org.fakekoji.xmlrpc.server.xmlrpcresponse;
+
+public class PackageId implements XmlRpcResponse<Integer> {
+
+    private final Integer packageId;
+
+    public PackageId(Integer packageId) {
+        this.packageId = packageId;
+    }
+
+    public PackageId(Object object) {
```

Comment:
Overall, I'm for removing all constructors accepting `Object` and create static factory for it.